### PR TITLE
Feat/ml tools

### PR DESCRIPTION
**Pull request by jgersak**
add deprecation warning for utils.py
add missing types
update test calls (enums)

### CODE REVIEW

File: ml_tools/eolearn/ml_tools/utils.py [hunk 1]
```diff
@@ -11,18 +11,30 @@
...
-    axes=None,
+    window: Any = (0,),
+    asteps: Optional[Any] = None,
+    wsteps: Optional[Any] = None,
+    axes: Optional[Any] = None,
     toend: bool = True,
 ) -> np.ndarray:
+    warn(
+        "Function `rolling_window` is going to be removed, you should use `numpy.lib.stride_tricks.sliding_window_view`"
+        " instead of it.",
```
Comment (zigaLuksic) [inline ml_tools/eolearn/ml_tools/utils.py:33]:
```suggestion
        "Function `rolling_window` will be removed in a future version, please use `numpy.lib.stride_tricks.sliding_window_view`"
        " instead.",
```

File: ml_tools/eolearn/tests/test_train_split.py [hunk 2]
```diff
@@ -63,13 +65,17 @@ def test_train_split():
         assert unique_counts[0] == expected_count
 
     # seed=2 should produce different result than seed=1
-    patch = TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bins, split_type="per_class")(patch, seed=2)
+    patch = TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bins, split_type=TrainTestSplitType("per_class"))(
```
Comment (zigaLuksic) [inline ml_tools/eolearn/tests/test_train_split.py:68]: since it's an Enum, you should use `TrainTestSplitType.PER_CLASS`

Comment (jgersak) [reply ml_tools/eolearn/tests/test_train_split.py]: done

File: ml_tools/eolearn/ml_tools/utils.py (hunk 1 above)
Comment (jgersak) [reply ml_tools/eolearn/ml_tools/utils.py]: done

**Pull request merged**